### PR TITLE
Add redis maintenance docs

### PR DESCRIPTION
**Pull request by zugao**
## Summary

* Short redis maintenance docs

## Checklist

- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog

### CODE REVIEW

**Review by wejdross — APPROVED** [review]

File: docs/modules/ROOT/pages/explanations/comp-functions/vshn-redis.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+= VSHN Redis Function-io
+
+The set of transformation go functions applied to a VSHN Redis composition.
+
+== Maintenance
+
+Redis maintenances supports only patch upgrades.
+There's no plan to add automatic minor or major upgrades due to usual breaking changes in newer Redis versions.
```
Comment (TheBigLee) [inline docs/modules/ROOT/pages/explanations/comp-functions/vshn-redis.adoc:8]: I would rephrase this:
```suggestion
There's no plan to add automatic minor or major upgrades due to possible breaking changes in newer Redis versions.
```

File: docs/modules/ROOT/pages/explanations/comp-functions/vshn-redis.adoc [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+=== AppCat Implementation
+
+In AppCat there's a cronjob deployed for each instance that is provisioned.
+This cronjob will trigger a Patch update `Job`.
+The latest patch version is being searched in the official https://hub.docker.com/_/redis[docker repository].
```
Comment (TheBigLee) [inline docs/modules/ROOT/pages/explanations/comp-functions/vshn-redis.adoc:14]:
```suggestion
The latest patch version is being looked up in the official https://hub.docker.com/_/redis[docker repository].
```

File: docs/modules/ROOT/pages/explanations/comp-functions/vshn-redis.adoc [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+If a newer patch version is found then Redis is updated.
```
Comment (TheBigLee) [inline docs/modules/ROOT/pages/explanations/comp-functions/vshn-redis.adoc:15]:
```suggestion
If a newer patch version is available, then Redis will be updated.
```

**Review by TheBigLee — APPROVED** [review]
